greenpithumb/humidity_sensor.py

```diff
@@ -0,0 +1,27 @@
+import dht11_exceptions
+
+class HumiditySensor(object):
+	"""Wrapper for humidity sensor."""
+
+	def __init__(self, dht11_result):
```

Comment:
I think it makes more sense for `HumiditySensor`'s contructor to take as  a parameter an instance of `dht11.DHT11` (similar to how the other classes take an `adc` instance). Then `HumiditySensor` is responsible for calling the instance's `read()` function and also handling the result.